driver-core/src/test/java/com/datastax/driver/core/QueryTracker.java

```diff
@@ -36,7 +36,7 @@
  * which hosts were queried.
  */
 public class QueryTracker {
-    static final String QUERY = "select * from test.foo";
+    public static final String QUERY = "select * from test.foo";
```

Comment:
This class is getting a bit ugly since we have the same method overloaded a few times, I may look into coming up with a better approach.